### DISCUSSION TITLE
How do I install this? (ELI5)

### DISCUSSION THREAD

**Discussion by beanbeanbug** [tomasklaen/uosc#904] [Q&A]
I installed MPV because other people recommended it as a video player. I want to be able to choose a subtitle file from a dropdown like with VLC. I am not a programmer. I have no idea what a remote script is. When I paste "Set-ExecutionPolicy RemoteSigned -Scope CurrentUser" into the command prompt as directed I get "'Set-ExecutionPolicy' is not recognized as an internal or external command,
operable program or batch file." I do not know what this is, and do not know what this means. Does anybody have a step-by-step explanation as to what I need to do to get this working that a toddler could follow?

Apologies for the trouble, and thanks.

**Comment by tomasklaen** [comment]
You need to paste the windows commands into a powershell terminal, not the old `cmd.exe`, which is what you seem to be doing.
